Cargo.toml

```diff
@@ -76,6 +77,7 @@ members = [
     "transports/tcp",
     "transports/uds",
     "transports/websocket",
-    "transports/wasm-ext"
+    "transports/wasm-ext",
+    "transports/quic",
```

Comment:
This is ordered alphabetically
(the dependencies normally should also be, but we it hasn't been enforced)